### ISSUE TITLE
The app package needs to be updated to follow YunoHost changes

### ISSUE THREAD

**Issue by kingemir** [scith/odoo_ynh#14]
The app is incompatible with YunoHost version 2.4.2: The app package needs to be updated to follow YunoHost changes

**Comment by scith** [comment]
Yes I agree... No time at the moment but I really wish to improve this package.
Also the main dev will take place here: https://github.com/YunoHost-Apps/odoo_ynh
Perhaps I would like to propose the installation of the community fork during install as well
